Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedMapModel.swift

```diff
@@ -26,13 +26,10 @@ class PreplannedMapModel: ObservableObject, Identifiable {
     
     init(preplannedMapArea: PreplannedMapAreaProtocol) {
         self.preplannedMapArea = preplannedMapArea
-        
-        // Kick off a load of the map area.
-        Task.detached { await self.load() }
     }
     
     /// Loads the preplanned map area and updates the status.
-    private func load() async {
+    func load() async {
```

Comment:
We could add a guard status != .loaded here

Reply:
Added conditional checks. Do you want me to write it in a switch-case (case .packaged, .downloaded: … default: …) instead?